www/source/partials/docs/_commands-cli.html.md.erb

```diff
@@ -405,6 +405,7 @@ Creates a symlink for a package binary in a common 'PATH' location
 
 **FLAGS** 
 
+    -f, --force      Overwrite existing symlinks
```

Comment:
We should use the term `binlinks` here--these constructs might not be implemented forever as symlinks